blog/settings.py

```diff
@@ -3,11 +3,10 @@
 import environ
 
 env = environ.Env(DEBUG=(bool, False),)
-root = environ.Path(__file__) - 3
-environ.Env.read_env()
-# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-BASE_DIR = root()
 
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = environ.Path(__file__) - 2
+env.read_env(str(BASE_DIR.path('.env')))
```

Comment:
Yes. I have the same problem as before
`UserWarning: /work/blog/.env doesn't exist - if you're not configuring your environment separately, create one.
  "environment separately, create one." % env_file)`